### DISCUSSION TITLE
Welcome to WhatsApp-Ai-bot Discussions!

### DISCUSSION THREAD

**Discussion by Zain-ul-din** [Zain-ul-din/whatsapp-ai-bot#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by btc2pkr** [comment]
share your idea

**Reply by Zain-ul-din** [reply]
Please see readme.md to know how the what's app bot is working. If you have any idea mention it in issues.
